script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_3.txt

```diff
@@ -101,10 +101,10 @@ Arihiko and I look at each other, furrowing our brows.
 "Well, it seems it's about time. I'm going back, but Tohno-kun, did you see the news this morning?"
 "Oh, ...I've been so busy I forgot. I haven't seen it yet."
 <Page31>
-"I see. Well then, let me ask you directly. There was a mansion on the news this morning. Was that your house by any chance?"
+"I see. Well then, let me ask you directly. There was a huge mansion on the news this morning. Was that your house by any chance?"
 "Eh?"
 <Page32>
-Now that she's said that, I remember Hisui saying that the police came over to question us.
+Now that she said that, I remember Hisui saying that the police came over to question us.

```

Comment:
I feel like `came by` reads a bit more naturally here, `came over` makes it sounds like the police are there for a playdate